pkg/controller/operators/olm/apiservices.go

```diff
@@ -184,17 +293,26 @@ func (a *Operator) installAPIServiceRequirements(desc v1alpha1.APIServiceDescrip
 	}
 	secret.SetName(apiServiceName + "-cert")
 	secret.SetNamespace(csv.GetNamespace())
+
+	// Add olmcasha hash as a label to the
+	caPEM, _, err := ca.ToPEM()
```

Comment:
just to verify: can we ever rotate the cert (ourselves) without updating the hash label? Just want to make sure we can't get into a failed state because of actions we made ourselves.